Dockerfile

```
@@ -1,6 +1,11 @@
-FROM golang:alpine3.15 as build
+FROM golang:1.19-alpine3.16 as build
```

Do we need to specify the alpine version? In `go-spacemesh` we just use:

```suggestion
FROM golang:1.19-alpine as build
```

We only need to make sure Go 1.19 is used.